opensrp-chw/src/lmh/assets/json.form/family_register.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
Cannot complete this form. Please confirm that all the forms can be filled and the data is processed

Reply:
Can you please specify on which field you get the error? I am able to submit the form and It starts populating at All families list too.